custom/icds_reports/static/icds_reports/js/spec/adhaar-beneficiary.directive.spec.js

```diff
@@ -212,14 +212,14 @@ describe('Adhaar Beneficiary Directive', function () {
         controller.userLocationId = 'test_id4';
         controller.location = {name: 'name4', location_id: 'test_id4'};
         controller.selectedLocations.push(
-            {name: 'name1', location_id: 'test_id1'},
-            {name: 'name2', location_id: 'test_id2'},
-            {name: 'name3', location_id: 'test_id3'},
-            {name: 'name4', location_id: 'test_id4'},
-            {name: 'name5', location_id: 'test_id5'},
-            {name: 'name6', location_id: 'test_id6'}
+            {name: 'name1', location_id: 'test_id1', user_have_access: 0},
```

Comment:
it seems like the same set of changes had to be applied to many different files here. is there any way to port this to a base class or something so that a future change wouldn't have to modify so many files?

Reply:
@czue we clean the code and we try to remove duplicates if we have time :) As you know angular controllers were cleaned, next step will be to clean other js files.